zerver/lib/event_schema.py

```diff
@@ -1630,7 +1630,7 @@ def check_update_message(
         ("type", Equals("update_message_flags")),
         ("op", Equals("remove")),
         ("operation", Equals("remove")),
-        ("flag", EnumType(["read", "starred"])),
+        ("flag", str),
```

Comment:
Tested in the dev environment and the `flag` field is not limited to these two strings.

Reply:
Yeah, the all-caps definitions in `AbstractUserMessage` -- I think it's {`ALL_FLAGS` - `NON_API_FLAGS`} minus the "unused flags".